test/graphql/attach_files_to_sample_test.rb

```diff
@@ -0,0 +1,210 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+# good case
```

Comment:
You can remove these 3 lines. The test descriptions are good enough

Reply:
Ah whoops that was just part of my pseudo for planning the test cases, missed those 3 lines in cleanup. Fixed now.